src/redistributors/morpho/Common.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity 0.8.23;
+
+library Common {
+    error OutOfBoundsForInt256();
+    error OutOfBoundsForUInt256();
+    error ZeroTotalShares();
+
+    uint256 internal constant PRECISION = 2 ** 128;
```

Comment:
explanation on the precision choice will be help here

Reply:
There is no explanation yet as it is the default and is arbitrary. We will need to do testing to arrive at an optimal value for our use-case.